packages/attention/src/component.tsx

```diff
@@ -185,13 +183,11 @@ export function Attention(props: AttentionProps) {
         {canClose && (
           <button
             type="button"
-            aria-label={i18n._(
-              /*i18n*/ {
-                id: 'attention.aria.close',
-                message: 'Close',
-                comment: 'Aria label for the close button in attention',
-              },
-            )}
+            aria-label={i18n._({
+              id: 'attention.aria.close',
+              message: 'Close',
+              comment: 'Aria label for the close button in attention',
+            })}
```

Comment:
I'm not sure this makes any difference in production or if it's picked up by Crowdin after all, but we could run `pnpm messages:extract && pnpm messages:compile` when updating translation logic in the components.

Reply:
Updated in these commits: [8e6c70c](https://github.com/warp-ds/react/pull/274/commits/8e6c70c66b1ac7e5e306cd55a0da250a9b765f85) and [3b29cbb](https://github.com/warp-ds/react/pull/274/commits/3b29cbb736f59b69888ec013f845e5ccbfc3e67d) 🙌